src/io/files.js

```diff
@@ -683,6 +683,68 @@ p5.prototype.loadXML = function() {
   return ret;
 };
 
+/**
+ * Loads a binary file from a file or a URL, and returns an Object
+ * with a bytes property containing a Uint8Array.
+ *
+ * This method is asynchronous, meaning it may not finish before the next
+ * line in your sketch is executed.
+ *
+ * @method loadBytes
+ * @param  {String}        path       name of the file or url to load
+ * @param  {function}      [callback] function to be executed after
+ *                                    loadBytes() completes, data is passed
+ *                                    in as first argument
+ * @param  {function}      [errorCallback] function to be executed if
+ *                                    there is an error, response is passed
+ *                                    in as first argument
+ * @return {Object}             contains a Uint8Array bytes property
+ * @example
+ *
+ * <p>Calling loadBytes() inside preload() guarantees to complete the
+ * operation before setup() and draw() are called.</p>
+ *
+ * <div><code>
+ * // TODO: Add Example with preload
+ * </code></div>
+ *
+ * <p>Outside of preload(), you may supply a callback function to handle the
+ * data:</p>
+ * <div><code>
+ * // TODO: Add Example with callback
+ * </code></div>
+ *
+ * @alt
+ * 50x50 ellipse that changes from black to white depending on the current humidity
```

Comment:
A note that this needs to be fixed as its leftover from the `loadJSON()` example.

Reply:
Probably need to be fixed, not sure when it was added but I think someone is currently working on adding alt text to canvases. I would say maybe edit it so it matches the example sketch or remove it for now.